patroni/ha.py

```diff
@@ -1066,7 +1072,8 @@ def restart(self, restart_data, run_async=False):
         self.set_start_timeout(timeout)
 
         # For non async cases we want to wait for restart to complete or timeout before returning.
-        do_restart = functools.partial(self.state_handler.restart, timeout, self._async_executor.critical_task)
+        do_restart = functools.partial(self.state_handler.restart, timeout, self._async_executor.critical_task,
```

Comment:
I don't think that we need to apply the stop timeout to the restart procedure.
If there is explicit restart command for a specific node that means we this node to retain its role. In case if nodes is the master not doing a clean shutdown will increase the downtime.